ArduPlane/quadplane.cpp

```diff
@@ -689,7 +686,7 @@ bool QuadPlane::setup(void)
         // this is a copter tailsitter with motor layout specified by frame_class and frame_type
         // tilting motors are not supported (tiltrotor control variables are ignored)
         if (tilt.tilt_mask != 0) {
-            gcs().send_text(MAV_SEVERITY_INFO, "Warning: Motor tilt not supported");
```

Comment:
To be clear, these send_text messages would never come out?

Reply:
right, I think QuadPlane::setup runs long before the connection to the gcs is active